### ISSUE TITLE
BatchNormalization layer not being folded when converting quantization-aware trained pb to tflite using tflite_convert

### ISSUE THREAD

**Issue by ruffles1** [tensorflow/tensorflow#25033]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Fedora 26
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): 1.13.0-dev20190118
- Python version: 3.6
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version: 10.0 / 7.4.1
- GPU model and memory: Nvidia Titan RTX

You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

**Describe the current behavior**
When converting a pb model file (that includes BatchNorm layers) to tflite, the resulting tflite graph continues to contain batchnorm layers in the form of mul and add ops.

**Describe the expected behavior**
Batch normalization ops should have been folded into weights and biases of previous layers in the tflite graph in order to optimize inference latency.

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by liyunlu0618** [comment]
Folded batchnorm has its limitations. Only certain ops are supported now:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/quantize/python/fold_batch_norms.py#L40
